src/components/InventoryManager/InventoryManagerScreen.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Can we remove the tan background color from the InventoryManagerScreen? We can just have it white for now